upgrade/upd-2.5.4-to-2.5.5/index.php

```
@@ -101,7 +101,7 @@ public function check_imptotal()
     public function apply_imptotal()
     {
         $sql = 'ALTER TABLE `' . $GLOBALS['xoopsDB']->prefix('banner') . "` CHANGE `imptotal` `imptotal` INT( 10 ) UNSIGNED NOT NULL DEFAULT '0'";
-        if (!$result = $GLOBALS['xoopsDB']->queryF($sql)) {
+        if (!$result = $GLOBALS['xoopsDB']->exec($sql)) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$result'. (undefined)

(UnusedLocalVariable)